perf/benchmark/runner/runner.py

```diff
@@ -185,8 +185,9 @@ def run(self, headers, conn, qps, size, duration):
             cacert_arg = "-cacert {cacert_path}".format(cacert_path=self.cacert)
 
         headers_cmd = ""
-        for header_val in headers.split(","):
-            headers_cmd += "-H=" + header_val + " "
+        if headers is not None:
```

Comment:
nit: `if headers:` is more idiomatic

Reply:
(1) `if headers` checks the value of headers, like [], None (or other zero-meaning values based on the data type of headers)
(2) `if headers is not None` compares `headers` with `None`. I think this is what we want, since we set `headers` ' default value to be `None`.